inc/options/general.php

```diff
@@ -438,166 +648,299 @@ class="w3tc-ignore-change" type="text"
-					<?php $this->checkbox( 'common.force_master' ) ?> <?php Util_Ui::e_config_label( 'common.force_master' ) ?></label>
-					<p class="description"><?php _e( 'Only one configuration file for whole network will be created and used. Recommended if all sites have the same configuration.', 'w3-total-cache' ); ?></p>
+					<?php $this->checkbox( 'common.force_master' ); ?> <?php Util_Ui::e_config_label( 'common.force_master' ); ?></label>
+					<p class="description"><?php esc_html_e( 'Only one configuration file for whole network will be created and used. Recommended if all sites have the same configuration.', 'w3-total-cache' ); ?></p>
 				</th>
 			</tr>
 			<?php endif; ?>
-			<?php if ( Util_Environment::is_nginx() ): ?>
+			<?php if ( Util_Environment::is_nginx() ) : ?>
 			<tr>
-				<th><?php Util_Ui::e_config_label( 'config.path' ) ?></th>
+				<th><?php Util_Ui::e_config_label( 'config.path' ); ?></th>
 				<td>
-					<input type="text" name="config__path" value="<?php echo esc_attr( $this->_config->get_string( 'config.path' ) ); ?>" size="80" <?php Util_Ui::sealing_disabled( 'common.' ) ?>/>
-					<p class="description"><?php _e( 'If empty the default path will be used..', 'w3-total-cache' ); ?></p>
+					<input type="text" name="config__path" value="<?php echo esc_attr( $this->_config->get_string( 'config.path' ) ); ?>" size="80" <?php Util_Ui::sealing_disabled( 'common.' ); ?>/>
+					<p class="description"><?php esc_html_e( 'If empty the default path will be used..', 'w3-total-cache' ); ?></p>
 				</td>
 			</tr>
 			<?php endif; ?>
 			<tr>
 				<th colspan="2">
-					<input type="hidden" name="config__check" value="0" <?php Util_Ui::sealing_disabled( 'common.' ) ?> />
-					<label><input type="checkbox" name="config__check" value="1"<?php checked( $this->_config->get_boolean( 'config.check' ), true ); Util_Ui::sealing_disabled( 'common.' ); ?> /> <?php Util_Ui::e_config_label( 'config.check' ) ?></label>
-					<p class="description"><?php _e( 'Notify of server configuration errors, if this option is disabled, the server configuration for active settings can be found on the <a href="admin.php?page=w3tc_install">install</a> tab.', 'w3-total-cache' ); ?></p>
+					<input type="hidden" name="config__check" value="0" <?php Util_Ui::sealing_disabled( 'common.' ); ?> />
+					<label><input type="checkbox" name="config__check" value="1" <?php checked( $this->_config->get_boolean( 'config.check' ), true ); ?> <?php Util_Ui::sealing_disabled( 'common.' ); ?> /> <?php Util_Ui::e_config_label( 'config.check' ); ?></label>
+					<p class="description">
+						<?php
+						echo wp_kses(
+							sprintf(
+								// translators: 1 opening HTML a tag to W3TC install admin page.
+								__(
+									'Notify of server configuration errors, if this option is disabled, the server configuration for active settings can be found on the %1$sinstall%2$s tab.',
+									'w3-total-cache'
+								),
+								'<a href="admin.php?page=w3tc_install">',
```

Comment:
Use `esc_url()` and `admin_url() for the `href`.

Reply:
Added esc_url and admin_url to href